packages/ui-react-utils/src/getInteraction.ts

```diff
@@ -34,14 +45,14 @@
  * @param {Object} args.props - the component props
  * @param {string} args.props.interaction - specifies the interaction mode, one of 'enabled', 'disabled', or 'readonly'
  * @param {boolean} args.props.disabled - specifies if the component is disabled. Will take precedence over readOnly
- * @param {boolean} args.props.readOnly - specifies if the component is readOnly
+ * @param {boolean} args.props.readonly - specifies if the component is readonly
```

Comment:
the name of the PROP is "readOnly", the interactionType option is "readonly"

Reply:
done